arch/eventloop_posix_tcp.c

```diff
@@ -490,6 +490,66 @@ TCP_registerListenSocketDomainName(UA_ConnectionManager *cm, const char *hostnam
     return UA_STATUSCODE_GOOD;
 }
 
+#ifdef _WIN32
+static
+UA_RegisteredFD *
+findRegisteredFD(TCPConnectionManager *cm, uintptr_t connectionId) {
+    UA_RegisteredFD *rfd, *rfd_tmp;
+    LIST_FOREACH_SAFE(rfd, &cm->fds, es_pointers, rfd_tmp) {
+
+        if (rfd->fd == connectionId) {
+            return rfd;
```

Comment:
I would say "explicit is better than implicit". Because it is opaque to the reader of the code that `LIST_FOREACH` assigns to `rfd` in every loop iteration. And that this assignment is still valid after exiting the loop.

Leaving it like this, it is more clear in my opinion, it also saves no lines of code.